### ISSUE TITLE
Pico Vimeo Tutorial Has No Video

### ISSUE THREAD

**Issue by leegold** [picocms/Pico#353]
https://vimeo.com/58496727

Has audio but no video instruction like the other vids.

**Comment by smcdougall** [comment]
Hate to tell you, but you're in the wrong place.  [Pico](http://picocms.org) has no connection to the software demonstrated in those videos.  Following a link on their Vimeo account will bring you [here](http://www.pico-cms.com/), where they have apparently rebranded their product some time ago.

At first glance, I would assume their software abandoned, since their landing page mentions the rebranding, but has no links besides a pdf manual.  I would probably suggest finding an alternative.

We would of course be happy to have you as a user of Pico, just keep in mind that we are not that software and have no affiliation with them.

**Comment by leegold** [comment]
I'd like to try your CMS but the documentation is too scant.

You need step by step cookbook example, just a couple of those...

**Comment by smcdougall** [comment]
We're working on it.  Honestly, lol.

Check out this work-in-progress [About Page](http://smcdougall.github.io/Pico/about/) if you'd like an overview of using Pico.  It should be enough for you to get an idea if Pico would be right for you or not.  The content of this page is done, it's just waiting on some updates to how the images are presented.

Just be aware that since it's a work-in-progress, it could break if I happen to be working on it when you check it out.

We also do have a "cookbook" in the works, but it's far from presentable at the moment.

And of course, if you do decide to give it a try, we're here to help with any issues you run into. :smiley:

**Comment by leegold** [comment]
What I'm doing is looking at: C:\wamp\www\pico\content-sample\index.md

in a good text editor. And it looks really clean! By that example I think I can make some headway.  I could actually use an editor and make snippets/clip list and syntax definition..

IMO if you had the simplest cookbook instructions on how to make a rudimentary site ... that's how I learn at least...

Do I ask simple new user question here? Is there a mailing list for the simpl,e probably obvious questions?

Thanks

**Comment by smcdougall** [comment]
> ...in a good text editor. And it looks really clean!

Your content files are formatted as "Markdown", which you can learn more about [here](http://daringfireball.net/projects/markdown/basics).  The syntax is simple enough that you don't need a special editor for it.  It was designed to be readable on its own.

That being said, there are many editors that support Markdown.  Personally, I use GitHub's [Atom](https://atom.io/) editor, but it's a bit overkill for just Markdown.  If you're going to do any other coding (HTML, CSS, Twig (available with a plugin)), I'd definitely recommend it though.

As far as designing your site, you can look at the [Default Theme](https://github.com/picocms/Pico/blob/master/themes/default/index.twig), located at `themes/default/index.twig`, for a good example.  It's a bit bare-bones, but it's intended to be a basis for designing your own layout.

Pico's "Themes" are actually Twig Templates.  Twig offers a lot of functionality for customizing your site.  You can learn about Twig more in-depth [here](http://twig.sensiolabs.org/), and our [Documentation](http://picocms.org/docs/#customization) gives an overview of what variables Pico uses to generate your site.

If that's a little too much for you though, we have a small, but growing selection of pre-made themes available on the [Pico website](http://picocms.org/customization/) and a few more available in the [wiki](https://github.com/picocms/Pico/wiki/Pico-Themes).

Personally, I develop a theme called [NotePaper](https://github.com/smcdougall/NotePaper) ([demo site](http://development.sjmcdougall.com/pico-themes/NotePaper/)), which offers quite a bit of customization out-of-the-box.  You could also use NotePaper or other themes as reference material for creating something of your own.

> IMO if you had the simplest cookbook instructions on how to make a rudimentary site

We really do have a [cookbook](http://picocms.org/cookbook/) started, but it hasn't gotten the attention it needs yet.  Pico was actually a stagnant project up until last fall when @PhrozenByte gave it some long needed improvements.

At the moment it's more-or-less the two of us.  I'm trying to slowly chip away at the documentation, and hopefully in a few months it's going to look entirely different.

> Do I ask simple new user question here?

You can check the [Getting Help](http://picocms.org/docs/#getting-help) section of our docs for more information, but to sum it up:  If you're unable to solve a problem you can create a [New Issue](https://github.com/picocms/Pico/issues) here on GitHub, or for that matter, you could continue this thread.  @PhrozenByte will probably close it at some point soon, but we'll still get notified if you post a comment.

You can also check out [#picocms on Freenode IRC](https://webchat.freenode.net/?channels=%23picocms).  @PhrozenByte is always logged in there and is usually willing to help.  He's away from home at the moment, so his ability to respond may be limited until sometime next week.

Unfortunately we don't have any mailing list, forum, or etc for simple questions (maybe in the future...), but we won't turn you away for asking them either. :wink:

Sorry if this was a lot to read... I do that. :unamused:

We're always excited to have new users though, so if you think Pico is something you'd like to stick with we're happy to have you. :smiley:
